src/functTest/groovy/com/bmuschko/gradle/docker/tasks/image/DockerBuildImageFunctionalTest.groovy

```diff
@@ -11,6 +11,8 @@ import spock.lang.Unroll
 
 class DockerBuildImageFunctionalTest extends AbstractGroovyDslFunctionalTest {
 
+    private String lastImageId
```

Comment:
This shouldn't be a class variable. It should be inlined into the test method.

Reply:
Ok I will move the variable inside the given: block of the test. But if I break the test in multiple tests, I will need the variable to be shared between tests. How can we manage to do that if the variable is not a class member?